src/styles/utils/_variables.scss

```
@@ -0,0 +1,15 @@
+$link-color: #00acdc;
+$button-color: rgba(0, 172, 220, 1);
+$color-white: white;
+$blocks-width: 166px;
```

No need to hardcode for the content, use 100% instead
```suggestion
$blocks-width: 100%;
```